pos_reset_search/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Point of Sale - Clear product search on click",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.0.1",
```

Comment:
FYI, this change is not necessary. (done automatically by bot, when merging).

Reply:
You're right, fixed.